apps/web/components/landing/Header.tsx

```
@@ -10,12 +10,15 @@ import { cn } from '@/lib/utils';
 
 export function Header() {
   const [isScrolled, setIsScrolled] = useState(false);
+  const [mounted, setMounted] = useState(false);
```

Biome (2.4.4)

[error] This variable mounted is unused.

(lint/correctness/noUnusedVariables)